### PR TITLE
Only show "development build" footer on debug releases

### PR DESCRIPTION
**Pull request by peppy**
Releases aren't release considered development builds any more.

### CODE REVIEW

File: osu.Desktop/Overlays/VersionManager.cs [hunk 1]
```diff
@@ -55,23 +57,30 @@ private void load(OsuColour colours, TextureStore textures, OsuGameBase game)
...
-                        {
-                            Anchor = Anchor.TopCentre,
-                            Origin = Anchor.TopCentre,
-                            Texture = textures.Get(@"Menu/dev-build-footer"),
-                        },
                     }
                 }
             };
+
+            if (DebugUtils.IsDebugBuild)
```
Comment (bdach) [inline osu.Desktop/Overlays/VersionManager.cs:64]: Thoughts about using `OsuGameBase.IsDeployedBuild` instead? I know you don't really run local releases - I sometimes do, but am not likely to get confused - but just throwing that out there as a potentially better suggestion.

Comment (peppy) [reply osu.Desktop/Overlays/VersionManager.cs:64]: That's the variable I was looking for.

**Comment by bdach** [comment]
On a quick hacky check, the version number is still around (disregard the red colour, it's because of debug build). Are we keeping that? I don't necessarily mind it staying, just figured I'd bring that up.

2021-01-06-153738_1920x1038_scrot

**Comment by peppy** [comment]
Leaving the build number is intentional for now, yep.

**Review by bdach — APPROVED** [review]

**Pull request merged**